### PR TITLE
Bug fix for gateway

### PR DESCRIPTION
**Pull request by kpango**
Signed-off-by: kpango <kpango@vdaas.org>

### Description:
bug fix of gateway for v1 release
#### 

### Environment:

- Go Version: 1.15.7
- Docker Version: 20.10.0-rc2
- Kubernetes Version: 1.20.2
- NGT Version: 1.13.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [x] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [x] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by kevindiu — APPROVED** [review]
LGTM.
Happy birthday 👏

**Review by hlts2 — APPROVED** [review]
LGTM 👍

**Comment by rinx** [comment]
I'm not sure but the several dependencies are updated in this PR. Is that okay?

**Comment by kpango** [comment]
@rinx I think it's okay, because each dependencies are small update and I check this build on my local cluster it works fine.

If you interest please check below each one looks not breaking changes and just update small things.
https://github.com/gocql/gocql/compare/9e3bb6cbd881...14aa133462a3
https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/compare/v0.13.4...v0.13.5
https://github.com/aws/aws-sdk-go/releases/tag/v1.37.0
https://github.com/lucasb-eyer/go-colorful/compare/v1.0.3...v1.2.0

**Review by rinx — APPROVED** [review]

**Comment by kpango** [comment]
@rinx Recently kubectl command may not find the resource even if you run a watch request immediately after deployment!
So I recently added the sleep command for scylla and it works.

example
https://github.com/vdaas/vald/blob/master/Makefile.d/k8s.mk#L261
https://github.com/vdaas/vald/blob/master/Makefile.d/k8s.mk#L282

**Comment by rinx** [comment]
yes, please.

**Comment by kpango** [comment]
okay, I'll do it.

**Comment by rinx** [comment]
Maybe E2E tests for deploying with Cassandra requires changes like this.
https://github.com/vdaas/vald/pull/899/commits/442e82d12c67fe6ed077510412c1cbdaf83c902f

**Comment by kpango** [comment]
@rinx thank you for your attention, I updated `.github/helm/values/values-scylla.yaml`

File: .github/workflows/e2e-deploy.yml [hunk 1]
```diff
@@ -110,8 +110,8 @@ jobs:
         run: |
           sudo apparmor_parser -R /etc/apparmor.d/usr.sbin.mysqld
 
-          make k8s/external/mysql/deploy
-          make k8s/external/redis/deploy
+          K8S_SLEEP_DURATION_FOR_WAIT_COMMAND=10 make k8s/external/mysql/deploy
+          K8S_SLEEP_DURATION_FOR_WAIT_COMMAND=10 make k8s/external/redis/deploy
```
Comment (rinx) [inline .github/workflows/e2e-deploy.yml:114]:
```suggestion
          make K8S_SLEEP_DURATION_FOR_WAIT_COMMAND=10 k8s/external/mysql/deploy
          make K8S_SLEEP_DURATION_FOR_WAIT_COMMAND=10 k8s/external/redis/deploy
```

File: .github/workflows/e2e-deploy.yml [hunk 2]
```diff
@@ -233,7 +233,7 @@ jobs:
           helm version
       - name: deploy vald
         run: |
-          make k8s/external/scylla/deploy
+          K8S_SLEEP_DURATION_FOR_WAIT_COMMAND=10 make k8s/external/scylla/deploy
```
Comment (rinx) [inline .github/workflows/e2e-deploy.yml:236]:
```suggestion
          make K8S_SLEEP_DURATION_FOR_WAIT_COMMAND=10 k8s/external/scylla/deploy
```

File: .github/workflows/e2e-deploy.yml [hunk 3]
```diff
@@ -341,7 +341,7 @@ jobs:
           helm version
       - name: deploy vald
         run: |
-          make k8s/external/minio/deploy
+          K8S_SLEEP_DURATION_FOR_WAIT_COMMAND=10 make k8s/external/minio/deploy
```
Comment (rinx) [inline .github/workflows/e2e-deploy.yml:344]:
```suggestion
          make K8S_SLEEP_DURATION_FOR_WAIT_COMMAND=10 k8s/external/minio/deploy
```

File: Makefile.d/k8s.mk [hunk 4]
```diff
@@ -201,6 +201,7 @@ k8s/vald/delete/scylla: \
 k8s/external/mysql/deploy:
 	kubectl apply -f k8s/jobs/db/initialize/mysql/configmap.yaml
 	kubectl apply -f k8s/external/mysql
+	sleep ${K8S_SLEEP_DURATION_FOR_WAIT_COMMAND}
```
Comment (rinx) [inline Makefile.d/k8s.mk:204]:
```suggestion
	sleep $(K8S_SLEEP_DURATION_FOR_WAIT_COMMAND)
```

File: Makefile.d/k8s.mk [hunk 5]
```diff
@@ -220,6 +221,7 @@ k8s/external/mysql/initialize:
 ## deploy redis to k8s
 k8s/external/redis/deploy:
 	kubectl apply -f k8s/external/redis
+	sleep ${K8S_SLEEP_DURATION_FOR_WAIT_COMMAND}
```
Comment (rinx) [inline Makefile.d/k8s.mk:224]:
```suggestion
	sleep $(K8S_SLEEP_DURATION_FOR_WAIT_COMMAND)
```

File: Makefile.d/k8s.mk [hunk 6]
```diff
@@ -258,12 +260,13 @@ k8s/external/cassandra/initialize:
 k8s/external/scylla/deploy: \
 	k8s/external/cert-manager/deploy
 	kubectl apply -f https://raw.githubusercontent.com/scylladb/scylla-operator/master/examples/common/operator.yaml
-	sleep 2
+	sleep ${K8S_SLEEP_DURATION_FOR_WAIT_COMMAND}
```
Comment (rinx) [inline Makefile.d/k8s.mk:263]:
```suggestion
	sleep $(K8S_SLEEP_DURATION_FOR_WAIT_COMMAND)
```

File: Makefile.d/k8s.mk [hunk 7]
```diff
@@ -279,7 +282,7 @@ k8s/external/scylla/delete: \
 ## deploy cert-manager
 k8s/external/cert-manager/deploy:
 	kubectl apply -f https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml
-	sleep 2
+	sleep ${K8S_SLEEP_DURATION_FOR_WAIT_COMMAND}
```
Comment (rinx) [inline Makefile.d/k8s.mk:285]:
```suggestion
	sleep $(K8S_SLEEP_DURATION_FOR_WAIT_COMMAND)
```

File: Makefile.d/k8s.mk [hunk 8]
```diff
@@ -295,6 +298,7 @@ k8s/external/cert-manager/delete:
 ## deploy minio
 k8s/external/minio/deploy:
 	kubectl apply -f k8s/external/minio
+	sleep ${K8S_SLEEP_DURATION_FOR_WAIT_COMMAND}
```
Comment (rinx) [inline Makefile.d/k8s.mk:301]:
```suggestion
	sleep $(K8S_SLEEP_DURATION_FOR_WAIT_COMMAND)
```

File: Makefile.d/k8s.mk [hunk 9]
```diff
@@ -306,6 +310,7 @@ k8s/external/minio/delete:
 ## deploy metrics-serrver
 k8s/metrics/metrics-server/deploy:
 	kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
+	sleep ${K8S_SLEEP_DURATION_FOR_WAIT_COMMAND}
```
Comment (rinx) [inline Makefile.d/k8s.mk:313]:
```suggestion
	sleep $(K8S_SLEEP_DURATION_FOR_WAIT_COMMAND)
```

**Review by rinx — CHANGES_REQUESTED** [review]

File: .github/workflows/e2e-deploy.yml (hunk 1 above)
Comment (kpango) [reply .github/workflows/e2e-deploy.yml:114]: I'll work on it after bug fix/

**Pull request merged**